src/main/java/uk/gov/companieshouse/pscfiling/api/model/filing/FilingData.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.companieshouse.pscfiling.api.model.filing;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class FilingData {
+
+    private String firstName;
+    private String otherForenames;
+    private String lastName;
+    private String dateOfBirth;
+    private String ceasedOn;
+    private String registerEntryDate;
+
+    @JsonCreator
+    public FilingData(@JsonProperty("first_name") String firstName,
```

Comment:
There are enough parameters here to use the builder pattern like `PscCommonDto` does